nana/modules/updater.py

```diff
@@ -53,44 +52,44 @@ async def initial_git(repo):
     os.rename('nana-old/nana/session/', 'nana/session/')
 
 
-@app.on_message(Filters.me & Filters.command("update", Command))
+@app.on_message(filters.me & filters.command("update", Command))
```

Comment:
Codacy found an issue: [updater is too complex (27) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)